dockerfiles/submitty/autograding-default/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apt-get install -qqy imagemagick
 ENV DRMEMORY_TAG release_2.0.1
 ENV DRMEMORY_VERSION 2.0.1-2
 ENV AnalysisTools_Version v.18.06.00
+ENV AnalysisToolsTS_Version v22.08.02
```

Comment:
This isn't necessarily something that has to be addressed in this PR, but I think it would be good to take a look at how we maintain these Docker files and develop a more robust way to keep them up-to-date.  As it is now, these Dockerfiles have to be updated each time a release is made in the AnalysisToolsTS repository, in addition to the version bump PR needed in the Submitty repository.  This isn't limited to AnalysisToolsTS either--version 2.0.1 was released in 2018 and the latest version of Dr. Memory is 2.5.

Reply:
It shouldn't be that hard to make a dependabot like GH action that runs every night or whatever, and checks if there's a newer version available for drmemory or analysis tools (or whatever other things we might want to check), and then makes an issue / PR to handle updating them. The biggest thing would be making sure our automated coverage around drmemory or whatever is good enough that we would catch any potential breaking change in its output or whatever.